src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -161,19 +163,19 @@ class SgeBackend extends Backend with SharedFileSystem with LazyLogging {
    * and returns the outputs for the call
    */
   private def pollForSgeJobCompletionThenPostProcess(backendCall: BackendCall, sgeJobId: Int): (ExecutionResult, Int) = {
-    val tag = makeTag(backendCall)
+    val logger = workflowLoggerWithCall(backendCall)
     val abortFunction = killSgeJob(backendCall, sgeJobId)
     val waitUntilCompleteFunction = waitUntilComplete(backendCall)
     backendCall.callAbortRegistrationFunction.register(AbortFunction(abortFunction))
     val jobReturnCode = waitUntilCompleteFunction
     val continueOnReturnCode = backendCall.call.continueOnReturnCode
-    logger.info(s"$tag SGE job completed (returnCode=$jobReturnCode)")
+    logger.info(s"SGE job completed (returnCode=$jobReturnCode)")
     val executionResult = (jobReturnCode, backendCall.stderr.toFile.length) match {
       case (r, _) if r == 143 => AbortedExecution // Special case to check for SIGTERM exit code - implying abort
       case (r, _) if !continueOnReturnCode.continueFor(r) =>
-        FailedExecution(new Exception(s"$tag SGE job failed because of return code: $r"), Option(r))
+        FailedExecution(new Exception(s"SGE job failed because of return code: $r"), Option(r))
```

Comment:
At other times you had to include the makeTag in this message (which admittedly I didn't like). Probably should be consistent at least